R/toolGetRegionCountries.R

```diff
@@ -0,0 +1,22 @@
+#' Get region mask that selects countries of a region based on a region mapping.
+#'
+#' @author Bennet Weiss
+#' @param x Magpie object
+#' @param region string or vector of strings to identify the region
+#' @param regionmapping mapping of regions to countries
+toolGetRegionMask <- function(x, region, regionmapping) {
+  mask <- magclass::getItems(x, dim = 1) %in% regionmapping$CountryCode[regionmapping$RegionCode %in% region]
+  return(mask)
+}
+
+#' Retrieve countries of a region from a region mapping.
```

Comment:
From the documentation of these two tool functions it is not clear to me how the parameter `x` is affecting the outcome.